packages/documentation-v7/src/stories/icons/components/icon.docs.mdx

```diff
@@ -0,0 +1,70 @@
+import LinkTo from '@storybook/addon-links/react';
+import { Meta, Canvas, Controls, Story, Source, ArgsTable } from '@storybook/blocks';
+
+import * as IconStories from './icon.stories';
+
+<Meta of={IconStories} />
+
+# Icon
+
+<p class="lead">

```

Comment:
```suggestion
<div class="lead">
```

MDX2 is already rendering texts within `<p>` tags. Using a `<div>` here will prevent having nested paragraphs (and get rid of a console error in the meantime!)